test/feature/docker-compose.yaml

```diff
@@ -13,6 +13,8 @@ services:
       interval: 10s
       timeout: 3s
       retries: 50
+    networks:
+      - featureTestNetwork
```

Comment:
it seems to me that it is more convinient to use snake_case. What do you think?

featureTestNetwork -> feature_test_network or feature-test-network